radis/io/tools.py

```diff
@@ -9,26 +9,28 @@
 import pandas as pd
 
 
-def parse_hitran_file(fname, columns, count=-1):
+def parse_hitran_file(fname, columns, count=-1, is_geisa=False):
```

Comment:
Yes, you can do a new function.

However it might not be needed if you manage the dtype=str trick below  https://github.com/radis/radis/pull/462#discussion_r868163618